src/module/get.js

```diff
@@ -0,0 +1,17 @@
+const container = document.getElementById('recent-scores-container');
+const getData = async () => {
+  const response = await fetch(
+    'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/lxpoC6G6d9W60ZW2y6PR/scores/',
+  );
+  if (!response.ok) {
+    throw new Error(`Something Went Wrong status: ${response.status}`);
+  }
+  const data = await response.json();
```

Comment:
- [ ] [Optional] You can decide to sort the result of the data gotten by arranging the scores from the highest to the lowest or the lowest to the highest. 👍